source/includes/critical-warning-text.rst

```diff
@@ -0,0 +1,4 @@
+The Critical Advisories below affect some prior ``mongosync``
+versions. If your deployment depends on features impacted by a
+Critical Advisory, upgrade to the latest available patch
```

Comment:
We shouldn't say "If your deployment depends on features impacted by a
Critical Advisory". The CA blast radius is wide enough that pretty much everyone needs to upgrade. Instead in the draft I suggested:

The Critical Advisories below affect some prior mongosync versions. Upgrade to the latest available patch release.